extensions/intellij/src/main/kotlin/dev/tigr/melody/plugin/Melody.bnf

```diff
@@ -0,0 +1,85 @@
+{
+  parserClass="dev.tigr.melody.plugin.parser.MelodyParser"
+
+  extends="com.intellij.extapi.psi.ASTWrapperPsiElement"
+
+  psiClassPrefix="Melody"
+  psiImplClassSuffix="Impl"
+  psiPackage="dev.tigr.melody.plugin.psi"
+  psiImplPackage="dev.tigr.melody.plugin.psi.impl"
+
+  elementTypeHolderClass="dev.tigr.melody.plugin.psi.MelodyTypes"
+  elementTypeClass="dev.tigr.melody.plugin.psi.MelodyElementType"
+  tokenTypeClass="dev.tigr.melody.plugin.psi.MelodyTokenType"
+
+  tokens=[
+    // keywords
+    of="of"
+    to="to"
+    capture="capture"
+    some="some"
+    match="match"
+    over="over"
+    option="option"
+    not="not"
+    either="either"
+    any="any"
+    ahead="ahead"
+    behind="behind"
+
+    // symbols
+    start="<start>"
+    end="<end>"
+    char="<char>"
+    whitespaceliteral="<whitespace>"
+    space="<space>"
+    newline="<newline>"
+    tab="<tab>"
+    return="<return>"
+    feed="<feed>"
+    null="<null>"
+    digit="<digit>"
+    vertical="<vertical>"
+    word="<word>"
+    alphabet="<alphabet>"
```

Comment:
@Tigermouthbear This changed to `alphabetic` recently and a few new symbols were added

Reply:
There are also a few more symbols added since these